### DISCUSSION TITLE
Understanding Error log

### DISCUSSION THREAD

**Discussion by olazzzzz** [freeplane/freeplane#1104] [Help] [answered]
Hi everyone,

Some weeks ago Freeplane started to show errors in my mindmaps. When I open the .log with Notepad++ , I see a whole lot of text which is quite difficult to follow for a non-expert. Nevertheless, in the midst of all those lines, I believe the problem to be "missing key OptionPanel.horizontal_layout".
Could anyone clarify what this is and how to solve it? By reinstalling the program?
Thank you!

**Comment by dpolivaev** [comment, marked as answer]
Create a bug report and attach your log file. It is thought to be used by developers for bug analysis.
